bayesian_cnn_prometheus/analysis/masks_analyzer.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import numpy as np
+
+from bayesian_cnn_prometheus.analysis.similarity_comparer import SimilarityComparer
+from typing import List
+
+
+class MasksAnalyzer:
+    def __init__(self, model_name: str, lesion_masks_path: str, variance_masks_path: str):
+        """
+        Creates MasksAnalyzer object to compare generated variance masks with respective lesion masks.
+        :param model_name: name of the model that has generated variance masks
+        :param lesion_masks_path: path to a file with original lesion masks
+        :param variance_masks_path: path to a file with generated variance masks
+        """
+        self.model_name = model_name
+        self.lesion_masks_path = lesion_masks_path
+        self.variance_masks_path = variance_masks_path
+        self.lesion_mask_names = self.load_mask_names(self.lesion_masks_path)
+        self.variance_mask_names = self.load_mask_names(self.variance_masks_path)
```

Comment:
Jeśli chcesz listę ścieżek plików to spróbuj glob.glob(PATH_TO_DIR) :)

Reply:
Z tym będzie problem - glob.glob() wykorzystuje pattern matching. Przydałyby się więc nazwa prefixu dla wyznaczoncyh masek wariancji, a widziałem, że nad tym pracuje @Qwebeck. Na razie zostawię zahardcodowaną wersję, poprawię jak wejdzie odpowiedni PR